src/app/screens/connect/btcSelectAddressScreen/index.tsx

```diff
@@ -0,0 +1,292 @@
+import BitcoinIcon from '@assets/img/dashboard/bitcoin_icon.svg';
+import stxIcon from '@assets/img/ledger/stx_icon.svg';
+import OrdinalsIcon from '@assets/img/nftDashboard/white_ordinals_icon.svg';
+import ActionButton from '@components/button';
+import SelectAccount from '@components/selectAccount';
+import useBtcAddressRequest from '@hooks/useBtcAddressRequest';
+import useWalletSelector from '@hooks/useWalletSelector';
+import { Check } from '@phosphor-icons/react';
+import { animated, useSpring } from '@react-spring/web';
+import { StickyHorizontalSplitButtonContainer } from '@ui-library/common.styled';
+import { getTruncatedAddress } from '@utils/helper';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+import { AddressPurpose } from 'sats-connect';
+import styled, { useTheme } from 'styled-components';
+import { getAppIconFromWebManifest } from './helper';
+
+const TitleContainer = styled.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  overflow: 'hidden',
+  marginLeft: 30,
+  marginRight: 30,
+  marginTop: 48,
+});
+
+const OuterContainer = styled(animated.div)({
+  display: 'flex',
+  flexDirection: 'column',
+  marginLeft: 16,
+  marginRight: 16,
+});
+
+const AddressBoxContainer = styled.div((props) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  width: '100%',
+  marginTop: props.theme.spacing(12),
+}));
+
+const AddressBox = styled.div((props) => ({
+  width: 328,
+  height: 66,
```

Comment:
don't think you should set an explicit width here, and if you do, it's a little too wide:
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/64b2149d-a024-461c-879a-0556e8644443)

Reply:
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/35254d2b-d05d-443d-a234-696bf158f0f5)

Reply:
and probably don't need an explicit height here either. it should follow this padding in figma:
![image](https://github.com/secretkeylabs/xverse-web-extension/assets/6109710/444fbeb9-58b0-4851-8ad2-fa6f60b14d92)

Reply:
explicit width was resolved, but the explicit height is still there. 

should use the correct padding instead
`padding: 20px 16px;`